### ISSUE TITLE
fetching numbers from twilio-cli fails

### ISSUE THREAD

**Issue by CLE4twilio** [twilio/twilio-cli#401]
### Issue Summary
I was trying to use an existing account and number in twilio quest and did succeed to load my account, but not to load a number to the game. that's why i tried to verify in twilio cli.... with the following result.

### Steps to Reproduce
1. This is the first step: install current version of twilio-cli
2. This is the second step: add a profile  in the environment variables
3.   check if account is configured corrctly with twilio profiles:list needs to show Active: true
4. List phone numbers associated to that account: twilio phone-numbers:list
---> fails with error below.

### Code Snippet
```node
# paste code here
```

### Exception/Log
```
PS C:\WINDOWS\system32> twilio phone-numbers:list -l debug
[DEBUG] Config File: C:\Users<myuser>.twilio-cli\config.json
[DEBUG] Using profile: ${TWILIO_API_KEY}/${TWILIO_API_SECRET}
[DEBUG] -- BEGIN Twilio API Request --
[DEBUG] GET https://api.twilio.com/2010-04-01/Accounts/AC<my account>/IncomingPhoneNumbers.json
[DEBUG] Custom HTTP Headers:
[DEBUG] User-Agent: twilio-cli/3.1.0 @twilio/cli-core/6.1.0 (win32 ia32) twilio-node/3.74.0 node/v16.13.2 phone-numbers:list
[DEBUG] -- END Twilio API Request --
[DEBUG] Found command "phone-numbers:list" plugin: twilio-cli
 » twilio-cli encountered an unexpected error. To report this issue, execute the command with the "-l debug" flag, then copy the output to a new issue here: "https://github.com/twilio/twilio-cli/issues"
[DEBUG] unable to get local issuer certificate
[DEBUG] Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1530:34)
    at TLSSocket.emit (node:events:390:28)
    at TLSSocket._finishInit (node:_tls_wrap:944:8)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:725:12)

### Technical details:
* twilio-cli version: 
* node version:
* Command output with debug logging enabled (adding `-l debug` to the end of the command):
```
# paste output here
```

**Comment by shrutiburman** [comment]
@CLE4twilio , thanks for reaching out.
Could you help me with the installation method that you used on your machine for `twilio-cli`? 
This is not an issue with the CLI as I couldn't reproduce it on installed twilio-cli exe, rather seems like an issue with the installation method.

**Comment by CLE4twilio** [comment]
Hi,I was using Windows packet install.Thanks Am 22.02.22, 19:28 schrieb shrutiburman ***@***.***>:

   
  @CLE4twilio , thanks for reaching out. Could you help me with the installation method that you used on your machine for twilio-cli? This is not an issue with the CLI as I couldn't reproduce it on installed twilio-cli exe, rather seems like an issue with the installation method.

**Comment by shrutiburman** [comment]
@CLE4twilio, I tried the packet install method too, this works fine on my machine. I don't see an error code here, any chance you're behind a proxy? 
You can try one of these options to authorise: 
1. Set the environment variable to extend pre-defined certs: `NODE_EXTRA_CA_CERTS`=path/to/your/cert.pem , you can find the certs using `find / -name *.crt`
OR
1. set `NODE_TLS_REJECT_UNAUTHORIZED=0` to bypass this check.

**Comment by CLE4twilio** [comment]
Thank you very much.This solved my problemAm 23.02.22, 15:53 schrieb shrutiburman ***@***.***>:

   
  @CLE4twilio, I tried the packet install method too, this works fine on my machine. I don't see an error code here, any chance you're behind a proxy? You can try one of these options to authorise: 
   
   Set the environment variable to extend pre-defined certs: NODE_EXTRA_CA_CERTS=path/to/your/cert.pem , you can find the certs using find / -name *.crt OR 
   set NODE_TLS_REJECT_UNAUTHORIZED=0 to bypass this check.

**Comment by shrutiburman** [comment]
Great @CLE4twilio ! I'll mark this as resolved then, please reach out via a new ticket, if you need further help.
